lib/tessel/deployment/rust.js

```diff
@@ -33,32 +37,134 @@ var exportables = {
         program
       };
     },
-    shell: () => {
+    shell: (opts) => {
       return tags.stripIndent `
         #!/bin/sh
-        exec /app/remote-script/rust_executable
+        exec /app/remote-script/${opts.resolvedEntryPoint}
```

Comment:
I think I made a mistake when I wrote "/app/remote-script/", because the paths are presently defined as:

```js
Tessel.REMOTE_PUSH_PATH = '/app/';
Tessel.REMOTE_RUN_PATH = '/tmp/remote-script/';
```